### PR TITLE
remove h1 tag

### PR DESCRIPTION
**Pull request by Ashbrdcn**

### CODE REVIEW

**Review by Osteicht — COMMENTED** [review]
Goods

**Comment by Ashbrdcn** [comment]
goods

File: route/public/homepage/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <body>
     <div class="body-section-container">
         <section class="header-container">
-            <h1>Hello Hello</h1>
+
```
Comment (Osteicht) [inline route/public/homepage/index.html:12]: check

**Review by Osteicht — COMMENTED** [review]
ok goods

**Pull request closed without merging**